core/src/main/scala/flatgraph/Graph.scala

```diff
@@ -156,7 +156,7 @@ class Graph(val schema: Schema, val storagePathMaybe: Option[Path] = None) exten
     this.closed = true
 
     storagePathMaybe.foreach { storagePath =>
-      logger.info(s"closing graph: writing to storage at `$storagePath`")
+      logger.info(s"closing graph: writing $nodeCount nodes and $edgeCount edges to storage at `$storagePath`")
```

Comment:
```suggestion
      logger.info(s"closing graph: writing $nodeCount nodes (and their edges) to storage at `$storagePath`")
```

We don't store an `edgeCount` anywhere - it's calculated on demand by walking the graph. 
Of course we could change that, but not sure if it's really super important for this log message.

Reply:
The idea is that we want to quickly spot from a log trace whether the graph loaded in separate runs was 'the same' or not. Ideally we'd output some sort of identity hash - but that's hard (and even more expensive to compute). We used to output nodes and edges in overflowdb, so that was my thought here too. So - edges aren't vital, but potentially still a useful signal.

It seems we can quite easily collect these counts one layer deeper during serialization. So I'll adjust this PR to do that, instead of walking the graph just for this log message.